bower.json

```diff
@@ -33,10 +33,10 @@
     "changelog.txt"
   ],
   "dependencies": {
-    "firebase": "2.x.x",
-    "rsvp": "3.1.x"
+    "firebase": "^2.4.0"
```

Comment:
Bower does not have the concept of peer dependencies, so this is still a regular dependency.